Framework/src/Aggregator.cxx

```diff
@@ -224,4 +224,13 @@ AggregatorConfig Aggregator::extractConfig(const core::CommonSpec& commonSpec, c
   };
 }
 
+void Aggregator::setActivity(std::shared_ptr<core::Activity> activity)
```

Comment:
You are not using this method anywhere, this PR will not make any difference as far as i can tell